Makefile

```diff
@@ -13,31 +13,69 @@
 # limitations under the License.
 
 GO := go
-
 EPOCH_TEST_COMMIT := f98810348b4b26d8c4dd64e36bde613049b2b0b9
+PROJECT := github.com/IBM/cap
+BUILD_DIR := _output
+SOURCES := $(shell find go/atom go/cap go/captn go/shared vendor/ -name '*.go')
```

Comment:
hurk, bleh
just let go figure it out. 1.10 has good caching now.

Reply:
can also set up travis to cache the go stuff as well.

Reply:
done..